developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables.md

```diff
@@ -50,7 +50,7 @@ The following environment variables are used for Jenkins setup.
 
 |Parameter| Description|
 |---------------|-----------|
-|github_release_tag | Specify a tag name here if you want to build from a tag. Example - release-2.7.0. This will look for a tag named release-2.7.0 in the repository URL configured in the Jenkins job and checkout the code from this tag. This should not be confused with **public_repo_branch**. The **public_repo_branch** is used only to checkout the Jenkinsfile which has all the build logic.|
+|github_release_tag | Specify a tag name here if you want to build from a tag. Example - release-2.8.0. This will look for a tag named release-2.8.0 in the repository URL configured in the Jenkins job and checkout the code from this tag. This should not be confused with **public_repo_branch**. The **public_repo_branch** is used only to checkout the Jenkinsfile which has all the build logic.|
 
 > **Note:** Even if the **public_repo_branch** is configured to a tag name, you need to provide a tag name in this parameter box when running the build. If this is empty, it will checkout code from the tag specified in **public_repo_branch** but it will not tag build artifact with the tag name. Instead it will tag it with commit hash which is undesirable when you want to build from tag. All build jobs create an artifact **metadata.json** that has details such as artifact/docker image name and version, and the Jenkins slave on which it was built.
```

Comment:
> **Note:** Even if the **public_repo_branch** is configured to a tag name, you need to provide a tag name in this parameter box when running the build. If this is empty, it will checkout code from the tag specified in **public_repo_branch**, but it will not tag build artifact with the tag name. Instead, it will tag it with commit hash which is undesirable when you want to build from the tag. All build jobs create an artifact **metadata.json** that has details such as artefact/docker image name, versions, and build version of the Jenkins slave.